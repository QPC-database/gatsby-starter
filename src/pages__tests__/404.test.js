import React from 'react'
import renderer from 'react-test-renderer'
import FourOhFour from '../pages/404'

describe(`404`, () =>
  it(`renders correctly`, () => {
    const tree = renderer.create(<FourOhFour />).toJSON()
    expect(tree).toMatchSnapshot()
  }),
)
