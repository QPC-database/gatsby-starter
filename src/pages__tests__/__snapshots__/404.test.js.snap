// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`404 renders correctly 1`] = `
<div>
  <h1>
    Page not found
  </h1>
  <p>
    You might find what you're looking for at 
    <a
      aria-current="page"
      className=""
      href="/"
      onClick={[Function]}
      onMouseEnter={[Function]}
      style={Object {}}
    >
      the index page
    </a>
    .
  </p>
</div>
`;
